source/lib/src/env_mat_nvnmd.cc

```diff
@@ -41,12 +41,10 @@ void deepmd::env_mat_a_nvnmd_quantize_cpu(std::vector<FPTYPE>& descrpt_a,
                                           const float& rmax) {
   // compute the diff of the neighbors
   rij_a.resize(sec_a.back() * 3);
-  fill(rij_a.begin(), rij_a.end(), 0.0);
+  fill(rij_a.begin(), rij_a.end(), (FPTYPE)0.0);
```

Comment:
Maybe not related to this PR - why does `fill` (`std::fill`?) work without `#include <algorithm>`?

Reply:
This item (`std::fill`) is also not included in `env_mat.cc`, but it is also work.

Reply:
Got it. Although I think there might be something wrong, I don't need it to be resolved in this PR.